portfolio/src/main/webapp/script.js

```diff
@@ -308,113 +323,109 @@ fetchReportMarkers() {
-  let directionsDisplay = new google.maps.DirectionsRenderer();
-  let request = {
-    origin: new google.maps.LatLng(31.742485, -106.485468),
-    destination: new google.maps.LatLng(31.774414, -106.501199),
-    provideRouteAlternatives: true,
-    travelMode: 'DRIVING'
-  }
-
-  let markerTry = new google.maps.LatLng({
-    lat: 31.761679,
-    lng: -106.491667
-  });
-
-  directionsDisplay.setMap(map);
-  directionsService.route(request, function(result, status) {
-    if (status === 'OK') {
-      console.log(result.routes.length);
-
-      /*for (var i =0; i < result.routes.length; i++) {
-          new google.maps.DirectionsRenderer({
-          map: map,
-          directions: result,
-          routeIndex: i
-          });
-      }*/
-
-      var counter = 0;
-
-      /*console.log(route.legs[0].steps.length);
-      console.log("Here we go again");
-      console.log(route.legs[0].steps[0].start_location.lat());
-      console.log(route.legs[0].steps[0].start_location.lng());
-      console.log(route.legs[0].steps[0].end_location.lat());
-      console.log(route.legs[0].steps[0].end_location.lng());
-      console.log(route.legs[0].steps[0].distance);*/
-      /*superMath(route.legs[0].steps[0].start_location, markerTry
-       * ,route.legs[0].steps[0].end_location);*/
-
-      var routeIndex = 0;
-      var lessCrimesInRoute = 1000;
-
-      for (var r = 0; r < result.routes.length; r++) {
-        var route = result.routes[r];
-
-        for (var j = 0; j < route.legs[0].steps.length; j++) {
-          var routeArray = new google.maps.Polyline({
-            path: [
-              new google.maps.LatLng(
-                route.legs[0].steps[j].start_location.lat(),
-                route.legs[0].steps[j].start_location.lng()),
-              new google.maps.LatLng(
-                route.legs[0].steps[j].end_location.lat(),
-                route.legs[0].steps[j].end_location.lng())
-            ]
-          });
-          for (var l = 0; l < markers.length; l++) {
-            var myPosition = new google.maps.LatLng(
-              markers[l].getPosition().lat(), markers[l].getPosition().lng());
-
-            if (google.maps.geometry.poly.isLocationOnEdge(
-                myPosition, routeArray, 0.0005)) {
-              console.log('Relocate!');
-              console.log(l);
-              console.log(route.legs[0].steps[j].end_location.lat());
-              console.log(route.legs[0].steps[j].end_location.lng());
-              counter++;
-            }
-          }
-        }
-
-        if (counter < lessCrimesInRoute) {
-          lessCrimesInRoute = counter;
-          routeIndex = r;
-          console.log(r);
-        }
-        counter = 0;
-      }
-
-      console.log(routeIndex);
-
-      new google.maps.DirectionsRenderer({
-        map: map,
-        directions: result,
-        routeIndex: routeIndex
-      });
+function route() {
+ 
+    var address = document.getElementById('d').value;
+    console.log(address);
```

Comment:
I see a lot of console.log(someVariable). This makes the console unreadable because if I look at the console I'll now see something like

1
54
7656
blah
foo
4645

If I wanted to know what 4645 meant, I would have to walk through the code line by line. Change ALL of these console.log to be more descriptive (`console.log("destination address: " + address)`) or remove them. There's too many for me to comment on each one. That way, I can at least do a search in the code to see where the print is coming from.